core/controllers/dashboard.py

```diff
@@ -262,6 +267,21 @@ def get(self):
         })
 
 
+class ExplorationDashboardStatsHandler(base.BaseHandler):
+    """Returns the new feedback for an exploration."""
+
+    @base.require_fully_signed_up
+    def get(self):
```

Comment:
Why not include exploration_id in the URL? That's standard with most of our handlers.